<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CARI_HESAP_EKSTRESI_BATCH</class>
 <widget class="QDialog" name="CARI_HESAP_EKSTRESI_BATCH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>357</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cari Hesap Ekstresi</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8" rowstretch="1,1,1,1,1,1,0,1" columnstretch="10,50,10,5">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkbox_hesap_araligi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Seçilen hesap aralığına göre cari hesap ekstresi almamızı sağlar.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QFrame" name="frame_hesap_araligi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="3,5,10,2">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Hesap Aralığı  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSearchEdit" name="searchedit_bas_hesabi">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Arama widget. </string>
        </property>
        <property name="QSearchEdit" stdset="0">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineedit_bas_hesap_ismi">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QSearchEdit" name="searchedit_bts_hesabi">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Arama widget. </string>
        </property>
        <property name="QSearchEdit" stdset="0">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineedit_bts_hesap_ismi">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkbox_tek_hesap">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Seçilen tek bir cari hesaba göre cari hesap ekstresi almamızı sağlar.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QFrame" name="frame_tek_hesap">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="3,5,10,2">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tek Hesap      </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineedit_tek_hesap_ismi">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSearchEdit" name="searchedit_tek_hesap">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Arama widget. </string>
        </property>
        <property name="QSearchEdit" stdset="0">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkbox_tarih_araligi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QFrame" name="frame_tarih_araligi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="5,7,1,7,1">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_tarih_araligi">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tarih Aralığı     </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QAdakDate" name="adakDate_bsl_tarihi">
        <property name="minimumSize">
         <size>
          <width>242</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>Tarih WidgetÄ± </string>
        </property>
        <property name="QAdakDate" stdset="0">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QAdakDate" name="adakDate_bts_tarihi">
        <property name="minimumSize">
         <size>
          <width>242</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>Tarih WidgetÄ± </string>
        </property>
        <property name="QAdakDate" stdset="0">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkbox_nakli_yekun">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Nakli yekün tutarına göre veya nakli yekün tutarı hariç rapor almamızı sağlar. </string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QFrame" name="frame_bakiyesiz_hesaplar_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_bakiyesiz_hesaplar_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Nakli Yekün</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkbox_bakiyesiz_hesaplar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>İşlem görmüş,borç alacak bakiye vermeyen cari hesapları raporda gösterir.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QFrame" name="frame_bakiyesiz_hesaplar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_bakiyesiz_hesaplar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Bakiyesiz Hesaplar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkbox_calismamis_hesaplar">
     <property name="toolTip">
      <string>İşlem görmemiş cari hesapları raporda gösterir.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QFrame" name="frame_calismamis_hesaplar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_calismamis_hesaplar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Çalışmamış Hesaplar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QFrame" name="frame_entegre_filtresi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="5,3,10">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_sirket">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Şirket             </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_sirket">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>582</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3">
    <widget class="QWidget" name="widget_batch_buttons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAdakDate</class>
   <extends>QWidget</extends>
   <header>QAdakDate.h</header>
  </customwidget>
  <customwidget>
   <class>QSearchEdit</class>
   <extends>QWidget</extends>
   <header>QSearchEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkbox_hesap_araligi</tabstop>
  <tabstop>searchedit_bas_hesabi</tabstop>
  <tabstop>searchedit_bts_hesabi</tabstop>
  <tabstop>checkbox_tek_hesap</tabstop>
  <tabstop>searchedit_tek_hesap</tabstop>
  <tabstop>checkbox_tarih_araligi</tabstop>
  <tabstop>adakDate_bsl_tarihi</tabstop>
  <tabstop>adakDate_bts_tarihi</tabstop>
  <tabstop>checkbox_nakli_yekun</tabstop>
  <tabstop>checkbox_bakiyesiz_hesaplar</tabstop>
  <tabstop>checkbox_calismamis_hesaplar</tabstop>
  <tabstop>comboBox_sirket</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
